src/js/services/integrations-service.js

```diff
@@ -37,17 +38,59 @@ export function resetTwilioAttributes() {
 export function fetchTwilioAttributes() {
     const {user: {clients, pendingClients}} = store.getState();
     const client = clients.find((client) => client.platform === 'twilio');
-    const pendingClient = pendingClients.find((client) => client.platform === 'twilio');
+    const pendingClient = pendingClients && pendingClients.find((client) => client.platform === 'twilio');
```

Comment:
I got an error at some point... but that's only because I was working with the wrong backend. `pendingClients` should always exist from now on.